WebComponent API Analyzer fails on Windows host

The scripts defined in the package.json chokes when executing on Windows host, both using Maven or running `npm run-script flow-components` on command line.

Here's the stacktrace:
```
[INFO] Running 'npm run-script flow-components' in c:\dev\github\vaadin-projects\flow\flow-components-parent\flow-webcomponent-api-analyzer
[INFO]
[INFO] > flow-webcomponent-api-analyzer@0.0.1 flow-components c:\dev\github\vaadin-projects\flow\flow-components-parent\flow-webcomponent-api-analyzer
[INFO] > bin/flow-webcomponent-api-analyzer.js --flow_components
[INFO]
[ERROR] 'bin' is not recognized as an internal or external command,
[ERROR] operable program or batch file.
[ERROR] npm ERR! code ELIFECYCLE
[ERROR] npm ERR! errno 1
[ERROR] npm ERR! flow-webcomponent-api-analyzer@0.0.1 flow-components: `bin/flow-webcomponent-api-analyzer.js --flow_components`
[ERROR] npm ERR! Exit status 1
[ERROR] npm ERR!
[ERROR] npm ERR! Failed at the flow-webcomponent-api-analyzer@0.0.1 flow-components script.
[ERROR] npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
```

From what I understand (and tested), using the node executable would make it work on all platforms rather than just OSX and Linux.

```
  "scripts": {
    ...
    "flow-components": "node bin/flow-webcomponent-api-analyzer.js --flow_components"
  },
```

--- a/flow-components-parent/flow-webcomponent-api-analyzer/package.json
+++ b/flow-components-parent/flow-webcomponent-api-analyzer/package.json
@@ -27,7 +27,7 @@
   },
   "scripts": {
     "flow-webcomponent-api-analyzer": "bin/flow-webcomponent-api-analyzer.js",
-    "flow-components": "bin/flow-webcomponent-api-analyzer.js --flow_components"
+    "flow-components": "node bin/flow-webcomponent-api-analyzer.js --flow_components"
   },
   "bin": {
     "flow-webcomponent-api-analyzer": "./bin/flow-webcomponent-api-analyzer.js"

